src/Redux/Members/constants.js

```diff
@@ -0,0 +1,12 @@
+export const GET_MEMBERS_PENDING = 'GET_MEMBERS_PENDING';
+export const GET_MEMBERS_SUCCESS = 'GET_MEMBERS_SUCCESS';
+export const GET_MEMBERS_ERROR = 'GET_MEMBERS_ERROR';
+export const POST_MEMBERS_PENDING = 'POST_MEMBERS_PENDING';
+export const POST_MEMBERS_SUCCESS = 'POST_MEMBERS_SUCCESS';
+export const POST_MEMBERS_ERROR = 'POST_MEMBERS_ERROR';
+export const PUT_MEMBERS_PENDING = 'PUT_MEMBERS_ERROR';
```

Comment:
there is an infinite spinner loop
```suggestion
export const PUT_MEMBERS_PENDING = 'PUT_MEMBERS_PENDING';
```

Reply:
donde